content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -87,51 +139,54 @@ To retrieve data from Filecoin using Lassie, all you need is the CID of the cont
     ...
     ```
 
-1. This will create an `output.car` file within your current directory:
+* `-o` is an optional flag that tells Lassie where to write the output to. If you don't specify a file, it will append `.car` to your CID and use that as the output file name.
 
-    ```shell
-    ls -l
-    ```
-
-    ```shell
-    total 143M
-    -rw-rw-r-- 1 user user 143M Feb 16 11:21 output.car
-    ```
+  If you specify `-`, as in our above example, the output will be written to stdout so it can be piped to another command, such as go-car, or redirected to a file.
 
-### Extract data
+* `<CID>/path/to/content` is the CID of the content you want to retrieve, and an optional path to a specific file within that content.
```

Comment:
would be nice to include an example of using this -- wikipedia is a good test -- @rvagg probably has some URLs he has tested with.

Reply:
Yeah, `bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze/wiki/Cryptographic_hash_function` (or pick your favourite page, perhaps `bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze/wiki/Lassie`).

`lassie fetch -o - bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze/wiki/Cryptographic_hash_function | car extract - | less` is supposed to work but is failing me at the moment and I don't know why, but if it works for you then feel free to include it.

Reply:
https://github.com/ipld/go-car/pull/409 fixes the problem of `car extract -` when you have an incomplete DAG (i.e. you have a specific path and just want to extract whatever it finds in there) coming from stdin. I'll cut a release as soon as that commit is approved and merged so just proceed here with the docs as if it works.

`go install github.com/ipld/go-car/cmd/car@2670bb87e2dc306b74dc5ca66ad763d24ad742a5` if you want to try it for yourself on that branch.

Reply:
It works for me with `go-car` commit. I am adding it to docs.